src/cli/index.js

```diff
@@ -315,7 +315,8 @@ function onUnexpectedError(err: Error) {
   const errorLoc = path.join(config.cwd, 'yarn-error.log');
   fs.writeFileSync(errorLoc, log.join('\n\n') + '\n');
 
-  reporter.error(reporter.lang('unexpectedError', errorLoc));
+  reporter.error(reporter.lang('unexpectedError', err.stack));
```

Comment:
Should I concat err.stack? It only shows the first part of the stack e.g: `Error: EACCES: permission denied, open '/home/jeffrey/Documents/Projects/yarn-testing/package.json'\n    at Error (native)` but I'm not sure how that works as it might result in the entire stack being printed?